tests/makeProgram.ts

```diff
@@ -2,30 +2,30 @@ import ts from "typescript";
 import path from "node:path";
 import fs from "node:fs";
 
-const operonRepoRoot = path.join(__dirname, "..");
+const sdkRepoRoot = path.join(__dirname, "..");
 
 function readFile(fileName: string) {
-  if (fileName.startsWith("node_modules/@dbos-inc/operon/")) {
-    const $path = path.join(operonRepoRoot, fileName.slice(30));
+  if (fileName.startsWith("node_modules/@dbos-inc/dbos-sdk/")) {
```

Comment:
This was poor code on my part originally. Can we extract `node_modules/@dbos-inc/dbos-sdk/` into a constant and then `.slice(constant.length)`?